dora/src/bytecode/generator.rs

```diff
@@ -811,6 +811,22 @@ impl<'a, 'ast> AstBytecodeGen<'a, 'ast> {
                     self.emit_invoke_direct(return_type, return_reg, fct_def_id, num_args);
                 }
             }
+            CallType::ModuleMethod(_, _, _) => {
+                let is_super_call = expr
```

Comment:
Do we really need to handle super and virtual calls here? What semantics does `ModuleName::methodName` have? IIUC this can't be super or virtual.

Reply:
Removed.